mystery_o_matic/output/latex/utils.py

```
@@ -0,0 +1,110 @@
+def get_enumeration_list(elements, name=""):
+    r = "\\begin{enumerate}\n"
+    for element in elements:
+        r += "\\item " + element + "\n"
+    r += "\\end{enumerate}"
+    return r
+
+def save_tex(outdir, language, tex):
+    try:
+        makedirs(outdir + f"/{language}")
+    except OSError:
+        pass
```

Ruff

Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(OSError)`

(SIM105)